templates/smaily-woocommerce-admin.php

```diff
@@ -356,6 +364,98 @@ class="smaily-toggle"
 			</table>
 		</div>
 
+		<div id="checkout_subscribe">
```

Comment:
The form feels a bit difficult to understand at first glance. Made a mock up of a simpler approach. What do you think?
![Screenshot from 2020-05-06 14-38-18](https://user-images.githubusercontent.com/6755852/81173373-9f69a500-8fa8-11ea-8160-d489175db754.png)

Reply:
What about this kind of approach?
![image](https://user-images.githubusercontent.com/22918359/81293084-651a0980-9075-11ea-993e-823d9a7c9a93.png)

Reply:
It could still be misleading - would it mean that checkbox is still displayed, but in a disabled state?

Reply:
I went with approach first suggested
![image](https://user-images.githubusercontent.com/22918359/81315038-79b8ca80-9092-11ea-8656-ac1f71862679.png)